packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/utils/hasContent.js

```diff
@@ -0,0 +1,17 @@
+import isEmpty from 'lodash/isEmpty';
+
+export default function hasContent(type, content, metadatas, fieldSchema) {
+  if (type === 'component') {
+    const {
+      mainField: { name: mainFieldName },
+    } = metadatas;
+
+    if (fieldSchema?.repeatable) {
+      return content.some(item => !isEmpty(item[mainFieldName]));
```

Comment:
So it means that my repeatable component does not have the mainField filled it won't be displayed in the LV?

Why not doing the isEmpty on the entire object?

Reply:
Because we need to display something in the dropdown, which is (for now) only the mainField of each item. In case no mainField value is set in any of the repeatable components, there is nothing to display ... Do you have something in mind, that we could display instead?

Reply:
I would go for the ID. This is also an issue on the relation drop down (edit view). Display nothing or a dash without giving any information about that entry is so confusing. WDYT?

![Screenshot 2022-02-09 at 13 43 12](https://user-images.githubusercontent.com/7756284/153203572-30858147-b3dd-4f49-a10f-ef7c58d8bddb.png)

Reply:
Done in c51772aa2932ebf2cfa47750d042deac35e3cd4f.

@HichamELBSI Is the issue in the relation dropdown related to that PR? If not, could you create an issue instead? 🙏🏽

Reply:
I didn't bring this up for you to fix, it was just to show another similar example.